### ISSUE TITLE
Vertex shader binding issue

### ISSUE THREAD

**Issue by desatur** [desatur/Future#1]
![image-316](https://github.com/user-attachments/assets/140c5821-fdc1-45c5-8cf1-39abb689dc17)
Probably related to new assimp model loader

**Comment by desatur** [comment]
Nevermind, problematic shader

Logs:
```ERROR::SHADER_COMPILATION_ERROR of type: FRAGMENT
ERROR: 0:4: 'location qualifier on input' : not supported for this version or the enabled extensions
ERROR: 1 compilation errors.  No code generated.


 -- --------------------------------------------------- --
ERROR::PROGRAM_LINKING_ERROR of type: PROGRAM
Program Link Failed for unknown reason.

 -- --------------------------------------------------- --
```

**Comment by desatur** [comment]
Todo: Update to OpenGL 4.6 now and to Vulkan 1.3 in the Future

**Comment by desatur** [comment]
Fixed in [c86f0ea](https://github.com/desatur/Future/commit/c86f0eae52d64395b6a0364017723365c5f7247f)
